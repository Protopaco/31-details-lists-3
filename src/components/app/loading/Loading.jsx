import React from 'react';

const Loading = () => (
    <img src="http://static.demilked.com/wp-content/uploads/2016/06/gif-animations-replace-loading-screen-2.gif" alt="loading" />
)

export default Loading;