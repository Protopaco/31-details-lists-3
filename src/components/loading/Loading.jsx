import React from 'react';
import styles from './Loading.css';

const Loading = () => (
    <div className={styles.frame}>
        <img src="http://static.demilked.com/wp-content/uploads/2016/06/gif-animations-replace-loading-screen-2.gif" alt="loading" />
    </div>
)

export default Loading;
