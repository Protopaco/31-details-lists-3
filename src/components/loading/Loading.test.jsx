import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom'
import { render, cleanup } from '@testing-library/react';
import Loading from './Loading';

describe('Loading component', () => {
    afterEach(() => cleanup());
    it('renders Loading', () => {
        const { asFragment } = render(
            <Router>
                <Loading />
            </Router>);
        expect(asFragment()).toMatchSnapshot();
    });
});
