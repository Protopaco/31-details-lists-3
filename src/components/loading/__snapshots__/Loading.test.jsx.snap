// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component renders Loading 1`] = `
<DocumentFragment>
  <div
    class="frame"
  >
    <img
      alt="loading"
      src="http://static.demilked.com/wp-content/uploads/2016/06/gif-animations-replace-loading-screen-2.gif"
    />
  </div>
</DocumentFragment>
`;
